### ISSUE TITLE
Figure out how to render geojson in time-series.pymd

### ISSUE THREAD

**Issue by metasim** [locationtech/rasterframes#285]
Folium works in the notebook... can we make it work for the documentation? At a minimum, we could capture the output from the notebook and insert it as static HTML content.

**Comment by vpipkt** [comment]
fun fact, simplifying the geometry is probably smart. Here is a quick look at the simplify param versus number of vertices in the multipolygon:

```
0 		 9217
1e-07 		 4711
1e-06 		 2908
1e-05 		 1568
0.0001 		 921
0.001 		 320
```

Those are in degrees so 0.001 is nominally 110m; Modis pixel size is ~465m so that's probably about appropriate.

**Comment by vpipkt** [comment]
In notebook the `display` of the shapely geometry object has a `_repr_svg_` method so

![image](https://user-images.githubusercontent.com/7798319/63277826-0a851700-c274-11e9-8ed4-5ac008ab0aca.png)

**Comment by vpipkt** [comment]
So far I'm just getting the cell output of the display object...

`<IPython.display.GeoJSON object>`

**Comment by metasim** [comment]
@vpipkt I wonder what would happen if you also registered that formatter as the handler for `text/markdown`... since injecting HTML into markdown often works when the output target is HTML, it may get passed through and work as expected.

**Comment by vpipkt** [comment]
💥 good idea. see also issue #302

**Comment by vpipkt** [comment]
Immediate issue fixed by #299 more improvements coming with #302
